docs-src/content/sessions/update.md

```diff
@@ -165,7 +177,11 @@ self.client.processCommandResult(token: token, commandResult: commandResult) { (
 {{< /tab >}}
 
 {{< tab "Java/Android" "java" >}}
-// Example coming soon!
+SessionInput input = SessionInput.newBuilder()
```

Comment:
Define `commandResult` here and demonstrate setting its properties, like the other language examples do.